config.template.js

```diff
@@ -29,7 +29,7 @@ const G3W_KEYS = {
 const G3W_ADMIN_PATH = '../g3w-admin/g3w-admin'; // path to G3W-ADMIN main code
 
 let conf = {
-  assetsFolder:  './assets',      // path to G3W-CLIENT assets folder
```

Comment:
think of a way to use this fallback value when developing a previous version of g3w-client library

Reply:
@Raruto we need to create a release 3.7 with this pull request and set inside a config template:

// backward compatibilities (v3.6.x)
if (version < "3.7.0") {
config.assetsFolder = './assets',

Reply:
yes, maybe it might useful to use the alpha tag (eg. 3.7.0-alpha.0) when we start working on a new milestone.

For example:

- https://github.com/Leaflet/Leaflet/tags
- https://github.com/Leaflet/Leaflet/releases
- https://stackoverflow.com/questions/39206082/npm-version-to-add-alpha-postfix